pints/_nested/__init__.py

```diff
@@ -667,6 +667,10 @@ def run(self):
 
         return self._m_posterior_samples
 
+    def sampler(self):
+        """ Returns sampler used by the controller. """
```

Comment:
What do you think of including this specific comment, which is similar to the sampler() docstring on MCMC controller?
```suggestion
        """Returns the underlying :class:`NestedSampler` object."""
```

Reply:
Good point. I'll incorporate this into my new PR about multinest.